desktop/release-notes.md

```diff
@@ -33,6 +33,60 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/ge
 
 ### New
 
+- Introduce the new Dev Environments CLI `docker dev`.
+- Docker Desktop can now be installed to any drive and folder using the `--installation-dir`. Partially addresses [docker/roadmap#94](https://github.com/docker/roadmap/issues/94).
+- Implemented enhanced-container-isolation feature as part of Docker Desktop's hardening-desktop effort (public link to be provided).
```

Comment:
```suggestion
- Added Enhanced Container Isolation and Settings Management as new security features for Docker Desktop.
```
I'd also move this bullet point to the top of the list. I'll make a note to add the links to the docs once everything has been published, otherwise the build checks will fail.